app/controllers/admin/users_controller.rb

```diff
@@ -8,6 +8,12 @@ def index
     @users = @search.result
   end
   
+  def show
+    @total_questions = Question.count
+    @total_topics = Topic.count
+    @users = User.all    
```

Comment:
if this is page with general information it should be in `#show` action 
the `dashboard` seems to be pretty correct naming